docs/shared/packages/next/next-config-setup.md

```diff
@@ -125,7 +125,8 @@ module.exports = async (phase, context) => {
 
   // If you have plugins that has to be added after Nx you can do that here.
   // For example, Sentry needs to be added last.
-  updatedConfig = require('@sentry/nextjs')(updatedConfig, { silent: true });
+  const { withSentryConfig } = require('@sentry/nextjs');
```

Comment:
I would have put this line in the top scope but I assume you probably don't want that.